pagopa/pn-frontend
# PN-11291 / PN-11327 - Create notification a11y

## Short description
Fix some a11y bugs on create notification section.

## List of changes proposed in this pull request
- Add `tabIndex` on `codice tassonomico` field
- Remove useless `tabIndex` from text/infobox
- Fix aria-label on trash icon of attachments of create notification 

## How to test
On PA go to new notification section and navigate with tab key, you should be able to visit only interactive elements (including tooltip). On step 3, add an attachments and use voiceOver to read the page and verify that the label of trash icon is "Rimuovi documento"
